code/modules/mob/living/simple_animal/bot/floorbot.dm

```diff
@@ -233,23 +192,15 @@
 			return
 
 		if(loc == target || loc == target.loc)
-			if(istype(target, /obj/item/stack/tile/plasteel))
-				eattile(target)
-			else if(istype(target, /obj/item/stack/sheet/metal))
-				maketile(target)
-			else if(istype(target, /turf/) && emagged < 2)
+			if(istype(target, /turf/) && emagged < 2)
 				repair(target)
 			else if(emagged == 2 && istype(target,/turf/simulated/floor))
 				var/turf/simulated/floor/F = target
 				anchored = 1
 				mode = BOT_REPAIRING
-				if(prob(90))
-					F.break_tile_to_plating()
-				else
-					F.ReplaceWithLattice()
+				F.ReplaceWithLattice()
 				audible_message("<span class='danger'>[src] makes an excited booping sound.</span>")
-				spawn(50)
-					amount ++
+				spawn(5

```

Comment:
God. Damn. It.

I literally change 1 number slightly before commiting and don't bother to test it because its a single var, and I fuck it up.